src/main/webapp/entityNotFoundPage.jsp

```diff
@@ -22,12 +22,30 @@
 
     <div id="frameBodyWrapper" class="container">
         <div class="row">
-            <div class="alert alert-warning col-md-4 col-md-offset-4">
+            <div class="alert alert-warning col-md-6 col-md-offset-3">
                 <img src="/images/puzzled.png"
                     style="float: left; margin: 0 10px 10px 0; height: 90px;">
                 <p>
-                    We could not locate what you were trying to access.<br><br>
-                    Make sure that the URL is correct, and you are a registered user of TEAMMATES. [<a href="/logout.jsp">Logout</a>]
+                    TEAMMATES could not locate what you were trying to access. <br><br>
+                    If you are a registered user you can go back to the <a href="/page/studentHomePage">home page</a>
```

Comment:
Inserting this sentence here breaks the link between the one above and the one below.